src/installer/image/installer.go

```diff
@@ -37,17 +45,57 @@ func GetDigest(uri string) (string, error) {
 }
 
 func NewImageInstaller(fs afero.Fs, props *Properties) installer.Installer {
+	// Create default transport
+	transport := http.DefaultTransport.(*http.Transport).Clone()
+
+	if props.Dynakube.HasProxy() {
+		proxy, err := props.Dynakube.Proxy(context.TODO(), props.ApiReader)
+		if err != nil {
+			log.Info("failed to get proxy from dynakube", "proxy", proxy)
+			return nil
+		}
+
+		proxyUrl, err := url.Parse(proxy)
+		if err != nil {
+			log.Info("invalid proxy url", "proxy", proxy)
+		}
+		log.Info("proxy spec", "proxy", proxy, "proxyURL", proxyUrl.String(), "proxyURL.Host", proxyUrl.Host, "proxyURL.Port()", proxyUrl.Port())
```

Comment:
I suppose we can print proxy password to log using `proxyUrl.String()`

Reply:
fixed in e278a5f2

Reply:
Sorry I didn't notice, the `proxy` string may also contain proxy password. `proxy` variable is the input and `proxyUrl` variable is the output of `url.Parse` function.

Reply:
removed proxy one in b546b575